lib/pager_duty/connection.rb

```diff
@@ -186,6 +188,12 @@ def initialize(token, token_type: :Token, url: API_PREFIX, debug: false)
     end
 
     def get(path, request = {})
+      # The run_request() method body argument defaults to {}, which is incorrect for GET requests
+      # https://github.com/technicalpickles/pager_duty-connection/issues/56
+      # NOTE: PagerDuty support discourages GET requests with bodies, but not throwing an ArgumentError to prevent breaking
+      #   corner-case implementations.
```

Comment:
I'd love to do a `logger.warn` here as I think a `raise(ArgumentError)` would be too aggressive and potentially breaking, but I don't think we have a logger.  I see some logging in the Faraday setup but I don't think that's easily usable.  Let me know if there is a logger or if we're okay throwing an ArgumentError if someone tries to do a HTTP GET request with a body.